test/SpecHelper.js

```diff
@@ -2,6 +2,25 @@ beforeEach(function() {
   /* Make expect function available in all tests. */
   window.expect = chai.expect;
 });
+/*
```

Comment:
Please make sure the * line up for comment blocks! We are working on standardizing our code